### ISSUE TITLE
5fold cross validation

### ISSUE THREAD

**Issue by zouying-sjtu** [KaiyangZhou/pytorch-vsumm-reinforce#26]
i find the 5FCV may have some problem. it is not the standard K-fold cross validation.

**Comment by Swati640** [comment]
Yes I think the same, Its taking the average of last 5 videos in test set and not all the 5 folds.
